bluesky/simple_scans.py

```diff
@@ -228,6 +228,23 @@ class _StepScan(_BundledScan):
 
     def __call__(self, motor, start, finish, intervals, time=None,
                  subs=None, **kwargs):
+        """Invoke the scan
+
+        Parameters
+        ----------
+        motor
+        start : number
+            The start point of the motion
+        finish : number
+            The finish point of the motion
+        intervals : int
+            The number of steps between `start` and `finish`
+        time : number
+            The acquire time of the detector(s)?
```

Comment:
It is set through the magical DWIM interface which seems to do what _noone_ wants.  For AD it sets the total acquisition time to this value by changing the exposure time (iirc) but for scalers just sets the count time.